plugins/modules/zos_operator_action_query.py

```diff
@@ -192,12 +192,37 @@ def run_module():
     )
 
     result = dict(changed=False)
-
     module = AnsibleModule(argument_spec=module_args, supports_check_mode=False)
     requests = []
     try:
         new_params = parse_params(module.params)
-        requests = find_required_request(new_params)
+
+        cmd_result_a = execute_command("d r,a,s")
+        if cmd_result_a.rc > 0:
+            module.fail_json(
+                msg="A non-zero return code was received while querying the operator.",
+                stdout=cmd_result_a.stdout,
+                stderr=cmd_result_a.stderr,
+                rc=cmd_result_a.rc,
+                stdout_lines=cmd_result_a.stdout.splitlines() if cmd_result_a.stdout else None,
+                stderr_lines=cmd_result_a.stderr.splitlines() if cmd_result_a.stderr else None,
+                cmd="d r,a,s",
+            )
+
+        cmd_result_b = execute_command("d r,a,jn")
+        if cmd_result_b.rc > 0:
+            module.fail_json(
+                msg="A non-zero return code was received while querying the operator.",
+                stdout=cmd_result_b.stdout,
+                stderr=cmd_result_b.stderr,
+                rc=cmd_result_b.rc,
+                stdout_lines=cmd_result_b.stdout.splitlines() if cmd_result_b.stdout else None,
+                stderr_lines=cmd_result_b.stderr.splitlines() if cmd_result_b.stderr else None,
+                cmd="d r,a,s",
```

Comment:
Shouldn't this be "d r,a,jn" to tell the user what command generated the error?  It looks too much like #209, above.

Reply:
Good catch, fixed thanks @richp405